.github/workflows/ci.yml

```diff
@@ -18,12 +18,7 @@ jobs:
     needs:
       - main
       - bun
-      - fastly
-      - node
       - deno
-      - workerd
-      - lambda
-      - lambda-edge
```

Comment:
Only need to collect coverage from the `main`, `bun`, and `deno` jobs as `bun run test` will include coverage for the each of `fastly`, `node`, `workerd`, `lambda`, and `lambda-edge`